<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>module_article</class>
 <widget class="QDialog" name="module_article">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 210);

QLabel
{
font: 13px bold;
}

QLineEdit {
  width: 100%;
 
  
  display: inline-block;
  border: 1px solid #ccc;
  border-radius: 4px;
  box-sizing: border-box;
}


</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane { /* The tab widget frame */
border-top: 2px solid #000000;
}
QTabWidget::tab-bar {
left: 0px; /* move to the right by 5px */
}
/* Style the tab using the tab sub-control. Note that it reads QTabBar not QTabWidget */
QTabBar::tab {
background: #dcdcdc;
border: 2px solid #535353;
border-bottom-color: #535353; /* same as the pane color */
border-top-left-radius: 7px;
border-top-right-radius: 7px;
min-width: 150px;
padding: 8px;
color: #000000;
font-size:15px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:selected, QTabBar::tab:hover {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #53a7fa, stop: 0.4 #55aaff, stop: 0.5 #52a5f8, stop: 1.0 #4f9fee);
}
QTabBar::tab:selected {
border-color: #9B9B9B;
border-bottom-color: #535353; /* same as pane color */
color: #dcdcdc;
font-size:15px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:!selected {
margin-top: 2px; /* make non-selected tabs look smaller */
}
QLineEdit {
  width: 100%;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #ccc;
  border-radius: 4px;
  box-sizing: border-box;
}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Gestion des articles</string>
    </attribute>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>450</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Id de l'article</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>450</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom de l'auteur</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_narticle">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_idS">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>440</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_auteur">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nom de l'article</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QTableView" name="Tab_Article">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>521</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
    </widget>
    <widget class="QLineEdit" name="lineEdit_type">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>type</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>202</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>trier par nom d'article</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>202</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>trier par nom d'auteur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>372</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Generez fichier pdf</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>391</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Mailing</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_type_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_type_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>39</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
      </property>
      <property name="text">
       <string>Selectionnez fichier pdf a envoyer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer À : </string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Gestion des emisions</string>
    </attribute>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>460</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Id de l'emision</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>460</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom de l'emission</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nomp">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_idSE">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>450</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nome">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom du presentateur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QTableView" name="Tab_Emissions">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>531</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_idE">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>trier par nom d'emision</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>trier par nom de presentateur</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>391</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Mailing</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_type_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_type_9">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_21">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
      </property>
      <property name="text">
       <string>Selectionnez fichier pdf a envoyer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoyer À : </string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_22">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>382</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Generez fichier pdf</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Historique</string>
    </attribute>
    <widget class="QTableView" name="Tab_historique">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>521</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>380</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>380</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Generez un pdf</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Alarme</string>
    </attribute>
    <widget class="QTableView" name="Tab_alarme">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>521</width>
       <height>381</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
    </widget>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>h:mm </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>452</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>470</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom de l'alarme</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>110</width>
       <height>23</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom de l'alarme</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>460</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	
	
	
	background-color: rgb(89, 169, 255);
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	/*padding: 10px;*/

}</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>59</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>59</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Heure</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
